.github/workflows/gh-pages.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
         with:
-          node-version: '16'
```

Comment:
This change breaks the build for GitHub Pages. https://github.com/3YOURMIND/kotti/actions/runs/3550513262/jobs/5963966183

I also see no evidence that this is a render.com problem. It seems to be happening in node@18 in general, at least on CI, for whatever reason.

On my local macOS Ventura system, I can’t reproduce this issue, but that might be because of openssl versions since this issue is related to those as far as I understand

Reply:
Please read the description

Reply:
@carsoli I read the description like 5 times already today.

Reply:
if it also breaks github pages to use node 16.16 then basically only node 16.16 (LTS) and node 18.12 (LTS) got the openssl package upgrades and it would explain why gh-pages breaks starting 16.16 and render breaks for 18.12 (latest LTS specified previously on `.nvmrc`)


My suggestion would be to basically only change the `.npmrc` to 16 then so that render works and keep everything as is because at this point getting the openssl package upgrades breaks both gh-pages and render

Reply:
if you look at the failure logs on render's dashboard u will see that a sub-dependency of render is the one that errors with node version 18

Reply:
you'd have to manually trigger on your branch, because the latest manual trigger I did was on the last commit on my branch (this one)

Reply:
> if you look at the failure logs on render's dashboard u will see that a sub-dependency of render is the one that errors with node version 18

I don’t understand why you come to the conclusion that it’s a dependency of render.

Reply:
My current understanding is that `webpack@4` is incompatible with `openssl@3`.

One of the major changes of `node@17` was upgrading from `openssl@1.1` to `openssl@3` <https://nodejs.org/es/blog/release/v17.0.0/#openssl-3-0>

`node@16` is no longer LTS as of 2022-10-18 which fits approximately with when Render first showed build failures.

So, `nvm`’s `lts` got updated to `18` around that time, which caused render to install `18` with `openssl@3` and thus failed the webpack build.

Reply:
did you check the logs

Reply:
Here’s a webpack statement and potential workarounds for this bug: <https://github.com/webpack/webpack/issues/14532#issuecomment-947525539>

Reply:
> the same logs you're looking at

I used the link from the PR description

Reply:
it is webpack, I just went back to render's dashboard, you're right.

Reply:
I'll attempt this output.hashFunction: "xxhash64" on this PR then, and keep the upgrades to 18 everywhere, and to lts on `.nvmrc` 

let's see

Reply:
@carsoli yeah, if we can get that workaround to work I think that’s the best solution. I’d suggest restoring `lts` wherever possible and pinning the rest to `18` so we actually get security updates